### PR TITLE
Updating card container documentation

### PR DESCRIPTION
**Pull request by heatherlarsen**
### :pushpin: Summary

If merged, this PR introduces adjustments to the `card` container component.

**Preview link:** https://hds-website-git-heatherlarsen-component-card-hashicorp.vercel.app/components/card

### :hammer_and_wrench: Detailed description

Changes include: 
- Adding images to support the guidelines
- Adding keywords
- Minor grammar and language flow updates
- Updating heading levels

### :link: External links

Jira ticket: [HDS-1351](https://hashicorp.atlassian.net/browse/HDS-1351)

***

### 👀 Reviewer's checklist:

- [ ] +1 Percy if applicable
- [ ] Confirm that PR has a changelog update via [Changesets](https://github.com/changesets/changesets) if needed

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

[HDS-1351]: https://hashicorp.atlassian.net/browse/HDS-1351?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by MelSumner — APPROVED** [review]
:shipit: (as an aside, I think we missed an opportunity for card and deck (of cards) here. 😆)

File: website/docs/components/card/index.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ links:
   figma: https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?node-id=2359%3A19245&t=XC8SUxxJOFHgqYzK-1
   github: https://github.com/hashicorp/design-system/tree/main/packages/components/addon/components/hds/card
 previewImage: assets/illustrations/components/card.jpg
+keywords: ['tile', 'container']
```
Comment (jorytindall) [inline website/docs/components/card/index.md:10]:
```suggestion
keywords: ['tile', 'container', 'box']
```
Might also add `box` here

File: website/docs/components/card/partials/guidelines/guidelines.md [hunk 2]
```diff
@@ -2,46 +2,58 @@
...
+Use a card container to help build more consistently styled cards.
 
-### Types
+## Types
 
-#### Static cards
+### Static cards
 
-A Static Card may have interactive elements within it, but is not itself actionable or interactive.
+Static cards may have interactive elements within them but are not actionable or interactive.
```
Comment (jorytindall) [inline website/docs/components/card/partials/guidelines/guidelines.md:11]:
```suggestion
Static cards may have interactive elements within them but are not actionable or interactive on their own.
```

**Review by jorytindall — APPROVED** [review]
Couple of small things :shipit:

File: website/docs/components/card/index.md (hunk 1 above)
Comment (KristinLBradley) [reply website/docs/components/card/index.md:10]: +1 I've called this kind of container a "box" in the past

**Review by KristinLBradley — APPROVED** [review]

**Pull request merged**